dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/HibernateRelationshipStore.java

```diff
@@ -197,6 +197,43 @@ public List<Relationship> getUidsByRelationshipKeys(List<String> relationshipKey
     return getQuery(hql, Relationship.class).setParameter("keys", relationshipKeyList).list();
   }
 
+  @Override
+  public List<RelationshipItem> getRelationshipItemsByTrackedEntity(UID trackedEntity) {
+    @Language("hql")
+    String hql =
+        """
+                from RelationshipItem ri
+                where ri.trackedEntity.uid = :trackedEntity
+                """;
+    return getQuery(hql, RelationshipItem.class)
+        .setParameter(TRACKED_ENTITY, trackedEntity.getValue())
```

Comment:
I have a dejavu 😅 I would use literals if we use templates that define the parameter without a constant as scanning `:trackedEntity` with `"trackedEntity" is easier than navigating to the constant and back. ok maybe an ide lets you hover but who wants to touch a 🐭 😂 .

Reply:
Too many rebases and cherry-picks on my side :grimacing: